### ISSUE TITLE
Running server with customised folder structor 

### ISSUE THREAD

**Issue by lpires** [swagger-api/swagger-node#452]
For the editor I am using your support following command:

>  swagger_swagger_fileName=server/api.yaml swagger project edit

But trying to run:
`
>swagger_swagger_fileName=server/api.json swagger project test
`
or 
`
>swagger_swagger_fileName=server/api.json swagger project test
`

This seems not to be supported by the api as I get 
`
(node:13071) UnhandledPromiseRejectionWarning: Unhandled promise rejection (rejection id: 1): SyntaxError: Unexpected token s in JSON at position 0
(node:13071) DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
`
This is cause by your function readEnvConfig where it is trying to convert into a json the file path.
Wandering if you supported ?

**Comment by nor-bu** [comment]
do you have a solution for this?

**Comment by DenisPopov15** [comment]
@nor-bu @lpires Do you found some solution for this?

**Comment by nor-bu** [comment]
Yes and no, i switched to swaggerize-express is much better then swagger-node for me.
